ui/apps/platform/cypress/helpers/vulnmanagement/entities.js

```diff
@@ -79,21 +79,21 @@ const opnameForEntities = {
     policies: 'getPolicies',
 };
 
-// Headings on entities pages: uppercase style hides the inconsistencies.
+// Headings on entities pages has sentence case for entity type.
 const headingPlural = {
-    clusters: 'clusters',
-    components: 'components',
-    'image-components': 'image components',
-    'node-components': 'node components',
+    clusters: 'Clusters',
+    components: 'Components',
+    'image-components': 'Image components',
+    'node-components': 'Node components',
     cves: 'CVES',
-    'image-cves': 'Image CVES',
-    'node-cves': 'Node CVES',
-    'cluster-cves': 'Platform CVES',
-    deployments: 'deployments',
-    images: 'images',
-    namespaces: 'namespaces',
-    nodes: 'nodes',
-    policies: 'policies',
+    'image-cves': 'Image cves', // TODO cves is inconsistent
+    'node-cves': 'Node cves', // TODO cves is inconsistent
+    'cluster-cves': 'Platform cves', // TODO cves is inconsistent
```

Comment:
CVE (all uppercase)

Reply:
Aha, I misread the fine print about lodash string functions:
* https://lodash.com/docs/4.17.15#capitalize is what I wrote:
    > Converts the first character of `string` to upper case and the remaining to lower case.
* https://lodash.com/docs/4.17.15#upperFirst is what I meant:
    > Converts the first character of `string` to upper case.